web/index.html

```diff
@@ -1435,18 +1435,29 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 		colors: '#FF5588',
 		heads: [
 			gaugeChart('Utilization', '12%', '', '#FF5588')
-		]
+		],
+		info: 'Percent of time the disk was doing something. Read from field 10.'
 	},
 
 	'disk.backlog': {
-		colors: '#0099CC'
+		colors: '#0099CC',
+		info: 'Milliseconds spent doing I/Os multiplied by the number of operations currently in progress. Read from field 11.'
 	},
 
 	'disk.io': {
 		heads: [
 			gaugeChart('Read', '12%', 'reads'),
 			gaugeChart('Write', '12%', 'writes')
-		]
+		],
+		info: 'Amount of data transfered to and from disk. Computed from fields 3 and 7.'
+	},
+
+	'disk.ops': {
+		info: 'Number of operations done on disk. The operations requested to the disk might be higher because of merged operations. Read from fields 1 and 5.'
```

Comment:
Completed disk I/O operations. Keep in mind the number of operations requested might be higher, since the Linux kernel is able merge adjacent to each other (see merged operations chart).